### ISSUE TITLE
Project does not compile

### ISSUE THREAD

**Issue by upeter** [lagom/ibm-integration-examples#10]
When compiling the project I get a compilation error in: HelloJmsReceiverActor I get:
```
Error:(10, 35) object HelloJmsSourceFactory is not a member of package com.example.hello.impl.jms
Note: trait HelloJmsSourceFactory exists, but it has no companion object.
import com.example.hello.impl.jms.HelloJmsSourceFactory.RunSource
```
There seems to be no RunSource class at all.

**Comment by richdougherty** [comment]
Apologies, this was my mistake. I've submitted a fix for this now: #11.

**Comment by richdougherty** [comment]
This should be fixed now, but let me know if you have any more issues.
